db.js

```diff
@@ -371,6 +376,12 @@ export class ActivityPubDB extends EventTarget {
   }
 
   async ingestNote (note) {
+    const isFollowed = await this.isActorFollowed(note.attributedTo)
```

Comment:
Does it work when you remove this check now?

Reply:
Uh ohh.
Not working, it's still ingesting the notes if I remove `isFollowed`.

Reply:
Set breakpoints inside `ingestNote` and see what it's parent is in the call stack. It might be getting invoked from a different place. Also check that it isn't being invoked from the `ingestActivty` call. If it is, check that our attributedTo check is working correctly by setting a breakpoint and checking the values make sense.

Reply:
Yeah it's `getNote`

Reply:
Maybe `getNote` just shouldn't be ingesting notes then?

I think we had that in there initially because we wanted to cache notes. We can remove it and leave the caching to the browser and only ingest notes from ingested collections.

Reply:
did this:
https://github.com/hyphacoop/reader.distributed.press/pull/21/commits/fce66f0c2844f7c98af83e4e4864cf9a5c83c8f5

we're using `getNote` in `post.js`